public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/NestedResourceTable.tsx

```diff
@@ -5,21 +5,21 @@ import { useStyles2 } from '@grafana/ui';
 
 import NestedRows from './NestedRows';
 import getStyles from './styles';
-import { Row } from './types';
+import { Row, RowGroup } from './types';
 
 interface NestedResourceTableProps {
-  rows: Row[];
-  selected: string[];
+  rows: RowGroup;
+  selectedRows: RowGroup;
   noHeader?: boolean;
-  onRowToggleCollapse: (row: Row) => void;
+  fetchNested: (row: Row) => Promise<void>;
```

Comment:
minor nit, but i think i would prefer a more... "api" like name that describes the event? Like requestNestedRows or even onExpandRow or something like that? idk. don't stress this one too much